components/Discussion/DiscussionProvider.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import DiscussionContextProvider from './context/DiscussionContextProvider'
+import Discussion from './Discussion'
+
+type Props = {
+  discussionId: string
+  meta?: any // TODO
+}
+
+const DiscussionProvider = ({ discussionId, meta }: Props) => {
+  return (
+    <DiscussionContextProvider discussionId={discussionId}>
+      <Discussion meta={meta} />
+    </DiscussionContextProvider>
+  )
+}
+
+export default DiscussionProvider
```

Comment:
if it renders the discussion component but no children it's no longer a provider – rename.